ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -402,6 +402,13 @@ def sum(
             out=out,
         )
 
+    def byteswap(self, inplace=False):
+        if inplace:
+            self.ivy_array.byteswap(True)
+            return self
+        else:
+            return self.ivy_array.byteswap()
```

Comment:
There is no `byteswap` method in the ivy api right now, so this frontend function would require a manual implementation using other ivy functions.